### DISCUSSION TITLE
Material for MkDocs Online Playground - codesandbox.io

### DISCUSSION THREAD

**Discussion by cemrehancavdar** [squidfunk/mkdocs-material#5205] [Show and tell]
I've created a CodeSandbox template for mk-docs material that enables you to experiment without the need to set up local environments. The template automatically installs mkdocs-material, initiates it, and provides a preview. You can access the template via this link: https://codesandbox.io/p/sandbox/mkdocs-material-codesandbox-gmlfeh.

If you wish to do so, you can add this template to the quick start section or create a new "quick try" section in your readme.md file. @squidfunk

**Comment by squidfunk** [comment]
Thanks for sharing! Interesting!

Maybe I'm too dumb to not see it, but how do I rebuild the preview when making changes to Markdown files or adding files?

**Reply by cemrehancavdar** [reply]
All you need is *fork* the template from here.
![resim](https://user-images.githubusercontent.com/50503448/225733097-2949aba1-ab5c-4734-8eda-43792a322e3c.png)
Then click or create a new .md and switch to md edit mode like below and save (ctrl + s) then volia, your magic in browser.
![resim](https://user-images.githubusercontent.com/50503448/225733607-64742dd8-79db-4ee2-a997-9be7d12f16ad.png)

**Reply by squidfunk** [reply]
Thanks! Sadly, forking is only possible when you're signed in, which requires you to create an account. I'm not sure I want to force users into creating new accounts to try something out, so I currently don't think it's a good idea to advertise it on our docs. If we can somehow provide a free only playground that doesn't have these restrictions, we'll add it in a minute!

**Reply by cemrehancavdar** [reply]
Right. I don't think any cloud editors would to it. Still, i will check it. A pyodide based one could solve it. Closing discussing for know.
